Fix SSL problem, string name, add test script.

Th0masL opened the pull request:
Since the files were not encoded in UTF-8, they were considered binary files, so we can't see the diffs correctly on git.

The first thing I did in this PR was to convert the files to UTF-8, so we can then see the diffs.

The other changes in this PR are:
- Fix a typo in the `UseSSL` variable that was preventing from doing SSL requests
- Rename the string `kernel32` to `Kernel32`, as it seems like MT5 is now complaining about that
- Added a simple test script that can show that things are working fine

You can see the changes in detail by looking at the commits.

Here is the output of the test script:
```bash
21:00:46.572	TestWebRequest2 (EURUSD,H1)	http://ifconfig.me/ip
21:00:46.572	TestWebRequest2 (EURUSD,H1)	++Webrequest2[INFO] Set Host to: ifconfig.me and Path: /ip with Port: 80 SSL=false Insecure=false
21:00:46.572	TestWebRequest2 (EURUSD,H1)	+Open Inet...
21:00:46.713	TestWebRequest2 (EURUSD,H1)	HTTP Response Code:200
21:00:46.713	TestWebRequest2 (EURUSD,H1)	HTTP Response:
21:00:46.713	TestWebRequest2 (EURUSD,H1)	194.106.XX.XX

21:00:48.701	TestWebRequest2 (EURUSD,H1)	https://ifconfig.me/ip
21:00:48.701	TestWebRequest2 (EURUSD,H1)	++Webrequest2[INFO] Set Host to: ifconfig.me and Path: /ip with Port: 443 SSL=true Insecure=false
21:00:48.701	TestWebRequest2 (EURUSD,H1)	-Close Session...
21:00:48.701	TestWebRequest2 (EURUSD,H1)	-Close Connect...
21:00:48.701	TestWebRequest2 (EURUSD,H1)	+Open Inet...
21:00:48.862	TestWebRequest2 (EURUSD,H1)	HTTPS Response Code:200
21:00:48.862	TestWebRequest2 (EURUSD,H1)	HTTPS Response:
21:00:48.862	TestWebRequest2 (EURUSD,H1)	194.106.XX.XX

21:00:48.862	TestWebRequest2 (EURUSD,H1)	-Close Session...
21:00:48.862	TestWebRequest2 (EURUSD,H1)	-Close Connect...
21:00:48.862	TestWebRequest2 (EURUSD,H1)	18 leaked strings left
```

Th0masL commented:
If you don't mind, I would also like to improve the layout of the script by applying `clang-format` to beautify the content a bit, but I'll do that in another PR.

sirtoobii commented:
Hi @Th0masL 

Thank you very much for your contribution - I'm happy to have a deeper look by the end of August as am currently on holiday :)

Th0masL commented:
Sounds good, enjoy your holiday :)

Th0masL commented:
Something not related to this PR:
I see those logs lines when I run the script, but I'm not sure where/how to fix them.

```
2023.08.29 02:00:55.127	16 leaked strings left
2023.08.29 02:00:55.127	1024 bytes of leaked memory
```

sirtoobii commented:
Dear @Th0masL 

Sorry for letting you wait, I've fixed the encoding issue and added a `.gitattributes` file which should make sure that we get nice diffs. Can you rebase?

> If you don't mind, I would also like to improve the layout of the script by applying clang-format to beautify the content a bit, but I'll do that in another PR.

Yes, that would be super nice.

> 2023.08.29 02:00:55.127	16 leaked strings left
2023.08.29 02:00:55.127	1024 bytes of leaked memory

As soon this PR is merged I can have a look at it :)

Th0masL commented:
Sadly it seems like there's something wrong with the new branch when I try to rebase my `master` branch.

I rebased and it shows the following files to have been modified, even if I haven't touched them after the rebase:

```bash
        modified:   Libraries/WebRequest2/InetHttp.mqh
        modified:   Libraries/WebRequest2/WebRequest2.mq5
```

I'm on Windows, and according to the following [stackoverflow post](https://stackoverflow.com/questions/5009096/files-showing-as-modified-directly-after-a-git-clone), it's possibly related to either end of line chars, or file permissions.

sirtoobii commented:
Well, there is still the option of what I call a (ugly) "manual rebase": 
- Copy away your changed files
- Check out the revision where your created the fork
- Pull upstream
- Replace the contents of the files you changed
- Force push to your fork

Th0masL commented:
I already did that, and it doesn't seems to help.

```bash
$ git config --list | grep auto
core.autocrlf=false
```
```
$ rm -Rf webrequest2/
$ git clone https://github.com/sirtoobii/webrequest2
Cloning into 'webrequest2'...
remote: Enumerating objects: 27, done.
remote: Counting objects: 100% (9/9), done.
remote: Compressing objects: 100% (8/8), done.
remote: Total 27 (delta 0), reused 8 (delta 0), pack-reused 18
Receiving objects: 100% (27/27), 71.44 KiB | 1.30 MiB/s, done.
Resolving deltas: 100% (3/3), done.

$ cd webrequest2/

$ git status
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   Libraries/WebRequest2/InetHttp.mqh
        modified:   Libraries/WebRequest2/WebRequest2.mq5

no changes added to commit (use "git add" and/or "git commit -a")
```

sirtoobii commented:
Can you try again?

sirtoobii commented:
Look good to me - however, a small detail: It would be super helpful if your test script is mentioned in the Readme (especially how to use it) :) And thank you very much for your contribution!

> I've not touched the ex5 files, as, from a security point of view, nobody should trust ex5 files compiled by someone else on git anyway :)

Yes, I agree - that's mainly because all my knowledge of software development back then came from a few hobby projects. I will remove (and exclude them) as soon this PR is merged

Th0masL commented:
I've added a small explanation in the readme :)
